### DISCUSSION TITLE
Limit number of tags in TagsInput

### DISCUSSION THREAD

**Discussion by ALSHIHAB** [filamentphp/filament#8582] [Help] [answered]
Limit number of tags in TagsInput to be - for example- not more than 5 tags. I've found how to limit the number of letters in each tag but couldn't see how to limit the number of tags.

**Comment by leandrocfe** [comment, marked as answer]
```php
->rules(['max:5'])
```

**Reply by ALSHIHAB** [reply]
Great. a bit related problem. the validation message is a bit wrong " The keywords field must not be greater than 5 characters. ". How to change it? such that the message will be "The keywords field must not be greater than 5 keywords."

**Reply by leandrocfe** [reply]
Maybe a [custom-rule](https://filamentphp.com/docs/3.x/forms/validation#custom-rules)?
